release.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Find all package.json files in subdirectories, except node_modules, examples and irma-frontend
+set -euxo pipefail
+standalone_packages=`find . -name "package.json" -not -path "*/node_modules/*" -not -path "*/examples/*" -not -path "*/irma-frontend/*"`
+set +euxo pipefail
```

Comment:
It seems like you only want to disable echoing (i.e. `-x`) of the echo's below, but by copypasting all of `euxo pipefail` each time you also disable error handling. It would be better if all scripts include `set -euxo pipefail` once on line 2, and if you want to disable echoing, just use `set +x` and `set -x`.

Reply:
I didn't think of that. That is a good idea.